.github/workflows/daily-build.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
           devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
+          githubAccessToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Can't we use the `packagePAT` it self? it's the same secret it seems

Reply:
I'm using this in the code... That's more readable...

Reply:
let's keep this for clarity.